source/deployment/sso-ldap.md

```diff
@@ -4,13 +4,15 @@ _Available in Enterprise Edition E10 and E20_
 
 ### Overview
 
-Active Directory/LDAP integration offers the following benefits:
+Active Directory is a service that stores authentication and authorization details of all users on your organization's network. Integrating your Active Directory with Mattermost means that users can log into Mattermost without having to create new credentials. User accounts are managed in AD/LDAP and changes are synced with the Mattermost server.
+
+Benefits of integrating AD/LDAP with Mattermost include:
 
 - **Single sign-on.** Users can sign in to Mattermost with their AD/LDAP credentials.
-- **Centralized identity management.** Mattermost accounts can display user information from AD/LDAP, such as first and last name, email, and username.
-- **Automatic account provisioning.** New Mattermost user accounts are automatically created the first time a user signs in with their AD/LDAP credentials on the Mattermost server.
```

Comment:
I'd keep the word "provisioning" instead of "creation". Keep the other changes in this line.